.github/workflows/docker.yml

```diff
@@ -10,10 +10,38 @@ on:
       - main
 
 jobs:
+  get-specs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install wget tool
+        run: |
+          sudo apt-get update
+          sudo apt-get install wget
```

Comment:
You don't need to pass `--yes` when running apt-get install commands in github actions?

Reply:
no it works like this